server/src/main/java/org/elasticsearch/action/index/IndexRequest.java

```diff
@@ -112,6 +112,8 @@ public class IndexRequest extends ReplicatedWriteRequest<IndexRequest> implement
 
     private Map<String, String> dynamicTemplates = Map.of();
 
```

Comment:
maybe add a comment here that `rawTimestamp` field doesn't need to be serialised, because it is only used on the node the coordinates the request.

Reply:
done

Reply:
👍